src/Session/sessions.py

```diff
@@ -0,0 +1,3 @@
+class Sessions:
+    def __init__(self, profile):
+        pass
```

Comment:
I thjnk the profile parameter should be None by default. And then in the constructor, I think you should have a varaiable called `self.profile` and if profile is None, you should create a new Profile object, otherwise you can load one using the profile name.

Reply:
I think you should also have a method to redirect commands and a test file

Reply:
Hm actually on second thought, i dont think the Session should take in profile as a parameter. But it should have a `self.profile` variable. You can ignore the other stuff I said about it. Just set it to None for now

Reply:
Ok actualllyyy on a third thought maybeee just have the constructor do nothing and just have three methods:
* redirect
* start
* end

I was kinda drawing out my thoughts here on how I think the classes should interact with each other
<img width="695" alt="Screen Shot 2022-11-21 at 2 14 05 PM" src="https://user-images.githubusercontent.com/26729179/203168993-069ee805-1a9e-4a93-a823-bd3eaa473d1a.png">

Reply:
Sorry this took long, I'm not sure if I understood your suggestions properly but I made some changes. Let me know what else needs changing!

Reply:
Almost! You can remove the `None` parameter for the instructor and the `redirect` command can have some parameter called `user_input` or something.

Reply:
Hope this works!